velox/vector/VectorSaver.h

```diff
@@ -64,7 +64,12 @@ std::string restoreStringFromFile(const char* FOLLY_NONNULL filePath);
 // Write the vector to a file. Contents would include the size of the list
```

Comment:
Shouldn't contents include the type of the elements as well?

Reply:
Good point, since this is a std::vector it did not cross my mind. We can perhaps leverage the type system for velox vectors but they wont cover the full spectrum of types that a std::vector can store like std::string instead of using string_view. Also, for our specific case it might be an overkill and perhaps we can think about extending this if it ends up getting re-used for other types.
What do you think?

Reply:
This method is confusing to me since I read 'vector' and think Velox Vector. I do see 'std', which makes me pause and confuses me further. Given the use case of serializing a list of numbers, it might be less confusing to add a method to serialize a Buffer.

Reply:
>  it might be less confusing to add a method to serialize a Buffer.

Not sure what you mean.
Do you mean we should just add a method to serialize the underlying storage buffer in the vector, that is, pointed to by vector.data() ? that would loose the context of enclosing this data within a vector when it is deserialized and that onus would then lie on the caller.